### ISSUE TITLE
Multiple email addresses

### ISSUE THREAD

**Issue by mgrundkoetter** [jeroendesloovere/vcard#34]
Right now it is only possible to set exactly one email address. The method name "addEmail" implies the possibility to set more than just one. The attribute based type of emails should be possible like it is with several other properties (like address, phone number, etc.).
To be backwards compatible the method could be changed so the default attributes are still used when the method is called. Only if the user provides a second parameter, the default value of "EMAIL;INTERNET" is overwritten.

**Comment by jeroendesloovere** [comment]
@mgrundkoetter Thanks for your reporting this issue.
This is true and needs to be solved in the future.
Feel free to post a Pull Request integrating multiple EmailAddress in an easy way.
Thanks :-D

**Comment by mgrundkoetter** [comment]
Yes, I will. Any hints where I can find the valid possibilities for the attributes? (like WORK;HOME;PREF;...) Which spec document are you using for the vcard?

**Comment by jeroendesloovere** [comment]
Hi, for VCard the spec is: [rfc6350](http://tools.ietf.org/html/rfc6350#section-10.1)

**Comment by mgrundkoetter** [comment]
I have opened a pull request with a simple solution for this. The "PREF" property is implemented the same way it has been for phone already (just PREF as attribute along with others like "WORK" and so on). The spec shows it slightly different (PREF=<INT>;). As I think the output should be consistent here, we could either take the simple solution which also keeps compatibility to the current implementation OR change it. But then not only for email, but also for phone.

Example created vial OSX contacts app looks like:
EMAIL;type=INTERNET;type=WORK;type=pref:john@doe.com

I noticed one more general issue while working on this topic. It is not possible to add 2 properties with the same attribute set (like 2 phone numbers with WORK attribute). The later call to addXYZ() will just overwrite the first one, as the attribute(s) is used as key for the internal array of values. Having multiple values with the same attribute seems to be valid in the format, so I guess this is another bug?

E.g. 2 entries like that are perfectly valid and created via OSX contacts app:
TEL;type=WORK;type=VOICE:+0123456
TEL;type=WORK;type=VOICE:+09876543
